app/styles/ui/toolbar/_toolbar.scss

```diff
@@ -25,13 +25,12 @@
     display: flex;
     flex-direction: row;
     flex-shrink: 0;
-
-    & > :last-child {
-      flex-grow: 1;
-    }
```

Comment:
This has been made redundant after adding `flex-grow: 1;` to `.toolbar-dropdown`.

Reply:
> This has been made redundant after adding `flex-grow: 1;` to `.toolbar-dropdown`.